pages/common/supervisorctl.md

```diff
@@ -0,0 +1,24 @@
+# supervisorctl
+
+> Supervisor is a client/server system that allows its users to control a number of processes on UNIX-like operating systems.
+> Supervisorctl is the command-line client piece of the supervisor which provides a shell-like interface.
+
+- Start/stop/restart a process:
+
+`supervisorctl {{start|stop|restart}} {{process name}}`
+
+- Start/stop/restart all processes in a group:
+
+`supervisorctl {{start|stop|restart}} {{group name}}:*`
+
+- Show last N **bytes** of process stderr:
```

Comment:
Why does "bytes" need to be in bold here ? Lets remove the asterisks unless there's a need. Also, I would prefer a concrete no. (like 100) rather than N. Makes things easier to grasp.